internal/executor/handle.go

```diff
@@ -19,6 +18,8 @@ import (
 	"github.com/meshplus/bitxhub/pkg/vm"
 	"github.com/meshplus/bitxhub/pkg/vm/boltvm"
 	"github.com/meshplus/bitxhub/pkg/vm/wasm"
+
+	"github.com/cbergoon/merkletree"
```

Comment:
go fmt?

Reply:
left a blank between the external and internal packages